### PR TITLE
Adding events for 440

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: draft/2022-04-27-this-week-in-rust.md [hunk 1]
```diff
@@ -94,11 +94,51 @@ Rusty Events between 2022-04-27 - 2022-05-25 🦀
...
+    * [**Mid-month Rustful**](https://www.meetup.com/RustDC/events/vdhxgsydchbwb/)
+* 2022-05-18 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
+    * [**Rust Study/Hack/Hang-out night**](https://www.meetup.com/Vancouver-Rust/events/nwcmpsydchbxb/)
+* 2022-05-24 | Berlin, DE | [OpenTechSchool Berlin](https://www.meetup.com/opentechschool-berlin/)
+    * [**Rust Hack and Learn**](https://www.meetup.com/opentechschool-berlin/events/284399980/)
 
 ### North America
 
-### Oceania
+* 2022-04-27 | Boston, MA, US | [Boston Rust Meetup](https://www.meetup.com/BostonRust/)
```
Comment (mariannegoldin) [inline draft/2022-04-27-this-week-in-rust.md:134]:
```suggestion
* 2022-05-27| New York, NY, US | [Rust NYC](https://www.meetup.com/Rust-NYC/)
    * [**Intro to Tantivy: a full-text search engine library written in Rust**](https://www.meetup.com/Rust-NYC/events/285257552/)
```

File: draft/2022-04-27-this-week-in-rust.md [hunk 2]
```diff
@@ -94,11 +94,51 @@ Rusty Events between 2022-04-27 - 2022-05-25 🦀
...
+    * [**Remote Book Club: Rust for Rustaceans Chapter Discussion**](https://www.meetup.com/RustPhilly/events/285282177/)
+* 2022-05-05 | Charlottesville, VA, US | [Charlottesville Rust Meetup](https://www.meetup.com/Charlottesville-Rust-Meetup/)
+    * [**Dealing with failure: producing and consuming Errors in Rust**](https://www.meetup.com/Charlottesville-Rust-Meetup/events/285078007/)
+* 2022-05-10 | Berlin, DE | [OpenTechSchool Berlin](https://www.meetup.com/opentechschool-berlin/)
+    * [**Rust Hack and Learn**](https://www.meetup.com/opentechschool-berlin/events/284399988/)
+* 2022-05-10 | Dallas, TX, US | [Dallas Rust](https://www.meetup.com/Dallas-Rust/)
+    * [**Second Tuesday**](https://www.meetup.com/Dallas-Rust/events/vqtjcsydchbnb/)
+* 2022-05-10 | Rostock, DE | [Altow Academy](https://www.meetup.com/altow-academy/)
+    * [**7. Rust Meetup Rostock**](https://www.meetup.com/altow-academy/events/283819127/)
+* 2022-05-10 | Seattle, WA, US | [Seattle Rust Meetup](https://www.meetup.com/Seattle-Rust-Meetup/)
```
Comment (mariannegoldin) [inline draft/2022-04-27-this-week-in-rust.md:117]:
```suggestion
* 2022-05-10 | Saarbrücken, DE | [Rust-Saar](https://www.meetup.com/Rust-Saar/)](https://www.meetup.com/Rust-Saar/)
    * [**Meetup: 21u16**](https://www.meetup.com/Rust-Saar/events/285483060/)
* 2022-05-10 | Seattle, WA, US | [Seattle Rust Meetup](https://www.meetup.com/Seattle-Rust-Meetup/)
```

File: draft/2022-04-27-this-week-in-rust.md [hunk 3]
```diff
@@ -94,11 +94,51 @@ Rusty Events between 2022-04-27 - 2022-05-25 🦀
 
 ### Virtual
 
-### Europe
+* 2022-04-27 | Stuttgart, DE | [Rust Community Stuttgart](https://www.meetup.com/Rust-Community-Stuttgart/)
+    * [**Rust-Meetup**](https://www.meetup.com/Rust-Community-Stuttgart/events/qwgrssydcgbkc/)
+* 2022-04-28 | Linz, AU | [Rust Linz](https://www.meetup.com/Rust-Linz/)
+    * [**Rust Meetup Linz - 21st Edition**](https://www.meetup.com/Rust-Linz/events/285248503/)
+* 2022-05-03 | Beijing, CN | [WebAssembly and Rust Meetup (Rustlang)](https://www.meetup.com/Wasm-Rust-Meetup/)
```
Comment (mariannegoldin) [inline draft/2022-04-27-this-week-in-rust.md:101]:
```suggestion
* 2022-05-02 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
    * [**Mob Programming: Rome Tools**](https://www.meetup.com/RustPhilly/events/kkbktsydchbdb/)
* 2022-05-03 | Beijing, CN | [WebAssembly and Rust Meetup (Rustlang)](https://www.meetup.com/Wasm-Rust-Meetup/)
```

File: draft/2022-04-27-this-week-in-rust.md [hunk 4]
```diff
@@ -94,11 +94,51 @@ Rusty Events between 2022-04-27 - 2022-05-25 🦀
...
+    * [**Monthly WasmEdge Community Meeting, a CNCF sandbox WebAssembly runtime**](https://www.meetup.com/Wasm-Rust-Meetup/events/jbfnrsydchbfb/)
+* 2022-05-03 | Buffalo, NY, US | [Buffalo Rust Meetup](https://www.meetup.com/Buffalo-Rust-Meetup/)
+    * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/Buffalo-Rust-Meetup/events/284996307/)
+* 2022-05-04 | Indianapolis, IN, US | [Indy Rust](https://www.meetup.com/indyrs/)
+    * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/285121667/)
+* 2022-05-04 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
+    * [**Remote Book Club: Rust for Rustaceans Chapter Discussion**](https://www.meetup.com/RustPhilly/events/285282177/)
+* 2022-05-05 | Charlottesville, VA, US | [Charlottesville Rust Meetup](https://www.meetup.com/Charlottesville-Rust-Meetup/)
+    * [**Dealing with failure: producing and consuming Errors in Rust**](https://www.meetup.com/Charlottesville-Rust-Meetup/events/285078007/)
+* 2022-05-10 | Berlin, DE | [OpenTechSchool Berlin](https://www.meetup.com/opentechschool-berlin/)
```
Comment (mariannegoldin) [inline draft/2022-04-27-this-week-in-rust.md:111]:
```suggestion
* 2022-05-09 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
    * [**Mob Programming: Rome Tools**](https://www.meetup.com/RustPhilly/events/kkbktsydchbmb/)
* 2022-05-10 | Berlin, DE | [OpenTechSchool Berlin](https://www.meetup.com/opentechschool-berlin/)
```

Comment (mariannegoldin) [inline draft/2022-04-27-this-week-in-rust.md:111]:
```suggestion
* 2022-05-09 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
    * [**Mob Programming: Rome Tools**](https://www.meetup.com/RustPhilly/events/kkbktsydchbmb/)
* 2022-05-10 | Berlin, DE | [OpenTechSchool Berlin](https://www.meetup.com/opentechschool-berlin/)
```

**Comment by mariannegoldin** [comment]
Added some event suggestions, otherwise LGTM :)

File: draft/2022-04-27-this-week-in-rust.md [hunk 5]
```diff
@@ -94,11 +94,51 @@ Rusty Events between 2022-04-27 - 2022-05-25 🦀
```
[the remark this answers is not in the document]
Comment (andrewpollack) [reply draft/2022-04-27-this-week-in-rust.md:94]: I've been operating on inclusive on the Wed, so I think it's good as is!

**Pull request merged**